steward_proto/proto/aave_v2_stablecoin.proto

```diff
@@ -1,68 +1,118 @@
+/*
+ * This is Steward's Strategy Provider API for the AaveV2StablecoinCellar.sol Cellar contract which can be found in this repo:
+ * https://github.com/PeggyJV/cellar-contracts
+ *
+ */
+
 syntax = "proto3";
 package steward.v1;
 
 option go_package = "/steward_proto";
 
+/*
+ * Represents a function call to the Aave V2 Stablecoin cellar
+ */
 message AaveV2Stablecoin {
+    // The function you wish to execute on the target cellar
     oneof function {
-        EnterStrategy enter_strategy = 1;
-        ReinvestWithAmount reinvest_amount = 2;
-        Reinvest reinvest = 3;
-        ClaimAndUnstakeWithAmount claim_and_unstake_amount = 4;
-        ClaimAndUnstake claim_and_unstake = 5;
-        Rebalance rebalance = 6;
-        AccruePlatformFees accrue_platform_fees = 7;
-        TransferFees transfer_fees = 8;
-        SetInputToken set_input_token = 9;
-        RemoveLiquidityRestriction remove_liquidity_restriction = 10;
-        Sweep sweep = 11;
+        // Represents function `accruePlatformFees()`: [link](https://github.com/PeggyJV/cellar-contracts/blob/b93b4393299aee35421f83ef4ca2689256e5b354/contracts/AaveV2StablecoinCellar.sol#L702)
```

Comment:
Did you manually add these? The code it's pointing to is point-in-time based on a commit...if say the arguments of the function change, you'd have to know that and update this comment by hand. Something to consider whether or not it's a valuable addition, perhaps just linking to the overall contract at the top would suffice.

Reply:
Yeah I was solving the other side of the problem which is linking to a specific line of the head of `main` which could change. Will link at the top.